client/my-sites/marketing/do-it-for-me/difm-landing.tsx

```diff
@@ -248,15 +248,24 @@ export default function DIFMLanding( {
 	const dispatch = useDispatch();
 
 	const productCost = useSelector( ( state ) => getProductCost( state, WPCOM_DIFM_LITE ) );
+	const tiers = useSelector( ( state ) => getProductPriceTierList( state, WPCOM_DIFM_LITE ) );
+
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
-	const hasPriceDataLoaded = productCost && currencyCode;
+	const hasPriceDataLoaded = productCost && tiers.length > 1 && currencyCode;
```

Comment:
nit: `productCost && tiers.length > 1 && ` Probably can be replaced with  the following selector

https://github.com/Automattic/wp-calypso/blob/42a251eb71e12037b6c20b1d2e64366e38ff5ea0/client/state/products-list/selectors/is-products-list-fetching.js#L6-L8

Might not have the same effect so better to test before committing.

Reply:
True! We changed this in https://github.com/Automattic/wp-calypso/pull/66080. But now that the experiment is over, we can revert https://github.com/Automattic/wp-calypso/pull/66080. But let's tackle this in a separate PR. 🙂